generic/release-checklist.md

```diff
@@ -0,0 +1,106 @@
+---
+# Template generated by https://github.com/coreos/repo-templates; do not edit downstream
+---
+
+# Release process
+
+The release process follows the usual PR-and-review flow, allowing an external reviewer to have a final check before publishing.
+
+## Requirements
+
+This guide requires:
+
+ * A web browser (and network connectivity)
+ * `git`
+ * [GPG setup][GPG setup] and personal key for signing
+ * Write access to this GitHub project
+
+## Release checklist
+
+These steps show how to release version `x.y.z` on the `origin` remote (this can be checked via `git remote -av`).
+Push access to the upstream repository is required in order to publish the new tag and the PR branch.
+
+:warning:: if `origin` is not the name of the locally configured remote that points to the upstream git repository (i.e. `git@github.com:coreos/{{ git_repo }}.git`), be sure to assign the correct remote name to the `UPSTREAM_REMOTE` variable.
+
+- make sure the project is clean and prepare the environment:
+  - [ ] `git clean -fd`
+  - [ ] `RELEASE_VER=x.y.z`
+  - [ ] `PREVIOUS_RELEASE_VER=x.y.z-1`
+  - [ ] `UPSTREAM_REMOTE=origin`
+
+- write release notes on a dedicated branch and tag it (the commit and tag will be signed with the GPG signing key you configured):
+  - [ ] `git checkout -b release-${RELEASE_VER}`
+  - [ ] write release notes in `docs/release-notes.md`. You can follow the following template:
+    ```
+    cat <<EOF > release-notes.txt
+    This is release ${NEWTAG} of console-login-helper-messages.
```

Comment:
`console-login-helper-messages` needs to be templated.

But also, this message is pretty verbose for its content.  Is this what we want?